### PR TITLE
Updating registry-image-widgets to resolve nav-tabs-pf bug

### LINKED ISSUE #1979: Service names can get cut off at certain viewport sizes.
Service name is cut off when looking in a category at just over 1200px.

@rhamilto 

screen shot 2017-08-28 at 3 34 15 pm

### LINKED ISSUE #2002: Collapse long lines in resources inside my screen
https://api.ci.openshift.org/console/project/ci/browse/config-maps/config

Some configs like the above have some nasty long lines and it would be great if we could collapse those inside the screen. It was also strange at start that I couldn't see the "See All" button and I had to scroll all the way through to the end.

### LINKED ISSUE #2010: Landing page nav-tabs-pf have layout issues caused by upstream repo
screen shot 2017-09-05 at 10 42 46 am
screen shot 2017-09-05 at 10 42 54 am

https://github.com/openshift/registry-image-widgets/blob/master/listing.less#L465-L482 is the culprit.  These rules are not properly scoped (they apply to all .nav-tabs-pf in console) and are overly specific (use of !important).

cc: @petervo

### PR DESCRIPTION
**Pull request by rhamilto**
Fixes #2010

screen shot 2017-09-06 at 5 22 11 pm

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -46,8 +46,8 @@
     "angular-moment": "1.0.0",
     "angular-utf8-base64": "0.0.5",
     "file-saver": "1.3.3",
-    "origin-web-common": "0.0.52",
-    "origin-web-catalog": "0.0.42"
+    "origin-web-common": "0.0.53",
+    "origin-web-catalog": "0.0.43"
```
Comment (spadgett) [inline bower.json]: #1987 bumps common and catalog (and is required due to API changes in common)

Comment (rhamilto) [reply bower.json]: Removing and updating.

**Comment by rhamilto** [comment]
@spadgett, updated.

**Comment by spadgett** [comment]
[merge][severity: bug]

**Pull request merged**